configs/base.yml

```diff
@@ -1,14 +1,15 @@
 SERVER:
   HOST: '127.0.0.1'
   PORT: 37000
+  QUEUE_TYPE: "rayqueue"
 
 RAY:
   RAY_CLUSTER_PORT: 30037
   LAUNCH_RAY_CLUSTER: True
 
 MANAGER:
-  DISABLE_FIXED_NODE_INIT_INSTANCE: False
-  DISABLE_INIT_INSTANCE_BY_MANAGER: False
+  DISABLE_FIXED_NODE_INIT_INSTANCE: True
+  DISABLE_INIT_INSTANCE_BY_MANAGER: True
```

Comment:
if the default migration backend is gloo, they both should be false.

Reply:
the default migration backend is rpc now.

Reply:
We can set default migration backend as 'gloo' and set these two default values as False.